### ISSUE TITLE
Realpath error with 1.63.0

### ISSUE THREAD

**Issue by oksimon95** [antonbabenko/pre-commit-terraform#336]
The latest version of pre-commit terraform throws an error due to lack of realpath command. Are there any new packages that need to be locally installed for pre-commit terraform to run successfully?

```
Terraform validate.......................................................Failed
- hook id: terraform_validate
- exit code: 1

/Users/.../.cache/pre-commit/repoln3cyzan/hooks/terraform_validate.sh: line 9: realpath: command not found
/Users/.../.cache/pre-commit/repoln3cyzan/hooks/terraform_validate.sh: line 11: ./_common.sh: No such file or directory
```

**Comment by oksimon95** [comment]
macOS Monterey 12.1

**Comment by antonbabenko** [comment]
I think you can install `coreutils` to get `realpath` (`brew info coreutils`) but that should not be required for this project to work.

**Comment by oksimon95** [comment]
Installing `coreutils` seems to have fixed the realpath error, but now I'm getting the same error as this other issue: https://github.com/antonbabenko/pre-commit-terraform/issues/337

**Comment by antonbabenko** [comment]
`realpath` should not be required. There are workarounds like this - https://stackoverflow.com/a/18443300/550451

**Comment by yermulnik** [comment]
That's coming from Bash 3. I don't it's worth supporting ancient versions of tools. @oksimon95 you'd better upgrade to a modern Bash as I mentioned in #337. IMHO.

**Comment by MaxymVlasov** [comment]
>realpath should not be required. There are workarounds like this - [stackoverflow.com/a/18443300/550451](https://stackoverflow.com/a/18443300/550451)

It is already in dependencies and was added by you 4 month ago :)
https://github.com/antonbabenko/pre-commit-terraform/pull/267/files

**Comment by nyc3-szheng** [comment]
the fixes in this are related to bash and coreutils 

Im running into this issue on v1.64 on different platform. 

I tried digging into the source of the error but im confused at where to start. I had pre-commit terraform fmt work correctly sometime so this was confusing when I ran into this issue. 

My coworker has also run into this issue so it is reproducible.

https://github.com/antonbabenko/pre-commit-terraform/issues/349
